example/lib/main.dart

```diff
@@ -21,8 +21,8 @@ class Home extends StatelessWidget {
   Widget build(BuildContext context) {
     return Column(
       children: [
-        Text(LocaleNames.of(context).nameOf('zzzz')),
-        Text(LocaleNames.of(context).nameOf('fr_CA')),
+        Text(LocaleNames.of(context)!.nameOf('zzzz').toString()),
```

Comment:
isn't nameOf always returning a string?

Reply:
[flutter_localized_locales.dart:23](https://github.com/guidezpl/flutter-localized-locales/blob/master/lib/flutter_localized_locales.dart#L23)

```dart
  String? nameOf(String localeString) => data[localeString];
```

So it can be `null`, sometimes.

Reply:
ah right, and Text needs non-null